docs/troubleshooting/keyboard.md

```diff
@@ -2,7 +2,7 @@
 
 #### Can I use a keyboard for the game?
 
-You can, however you will need a keymapper since the game doesn’t have a built-in keyboard support.
+No, you cannot use a keyboard for the game.
```

Comment:
IMO keymapping is still a viable method, especially if you are playing on emulator (idk ask Mech about this)

Reply:
You need an emulator to do that. Otherwise, the game will flag the keymapper for requesting gesture accessibility permission.